python/hsfs/engine/spark.py

```diff
@@ -185,13 +201,7 @@ def save_dataframe(
             )
 
     def _save_offline_dataframe(
-        self,
-        table_name,
-        feature_group,
-        dataframe,
-        save_mode,
-        operation,
-        write_options,
+        self, table_name, feature_group, dataframe, save_mode, operation, write_options,
```

Comment:
looks odd to me that black changed this back.

Reply:
I can run black again, but I don't think I changed it manually.

Reply:
black changed it.